File: 02_Sort_Application/Level2/11004.py
# 보물
# https://www.acmicpc.net/problem/11004
# 힌트
# 1. 정렬 후 K-1 인자를 반환해주면 된다.
import sys


if __name__ == "__main__":
    N, K = map(int, sys.stdin.readline().split())

    a_list = list(map(int, sys.stdin.readline().split()))
    a_list.sort()

    print(a_list[K-1])

File: 02_Sort_Application/Level2/11004.java
// 보물
// https://www.acmicpc.net/problem/11004
// 힌트
// 1. 정렬 후 K-1 인자를 반환해주면 된다.
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;
import java.util.StringTokenizer;

public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		StringTokenizer st = new StringTokenizer(br.readLine());
		int N = Integer.parseInt(st.nextToken());
		int K = Integer.parseInt(st.nextToken());
		
		int[] A = new int[N];
		
		st = new StringTokenizer(br.readLine());
		for (int i = 0; i < N; i++) {
			A[i] = Integer.parseInt(st.nextToken());
		}
		
		Arrays.sort(A);
		System.out.println(A[K-1]);
	}
}
